core/documents-manager.php

```diff
@@ -174,14 +174,20 @@ public function register_document_type( $type, $class ) {
 	 * @return false|Document Document data or false if post ID was not entered.
 	 */
 	public function get( $post_id, $from_cache = true ) {
+
```

Comment:
Same for this file.

Reply:
* Sorry, I accidentely merged some other files while working late *